.simplecov

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# @todo: Always run simplecov again once
+# https://github.com/colszowka/simplecov/issues/404,
+# https://github.com/glebm/i18n-tasks/issues/221 are fixed
+if ENV['COVERAGE'] == 'true'
+  SimpleCov.start do
+    add_filter 'spec/rails/'
+  end
```

Comment:
@deivid-rodriguez , can please explain in more details why  adding filter with 'spec/rails/' needed?